rotkehlchen/tests/api/blockchain/test_evm.py

```diff
@@ -607,4 +607,7 @@ def test_adding_safe(rotkehlchen_api_server: 'APIServer') -> None:
     result = assert_proper_sync_response_with_result(response)
     assert result == {
         'added': {safe_address: ['arbitrum_one', 'base']},
+        'failed': {
+            '0x9d25AdBcffE28923E619f4Af88ECDe732c985b63': ['binance_sc'],
```

Comment:
why this change?

Reply:
the api was failing during the re-decoding but it works now, reverted the change